src/raw_logging.cc

```diff
@@ -59,7 +59,7 @@
 # include <unistd.h>
 #endif
 
-#if defined(HAVE_SYSCALL_H) || defined(HAVE_SYS_SYSCALL_H)
+#if !defined(ANDROID) && (defined(HAVE_SYSCALL_H) || defined(HAVE_SYS_SYSCALL_H))
```

Comment:
I really think we should not use private APIs.
It's safer to modify errno instead of using private API.